frontend/explorer/public/js/main.js

```diff
@@ -119,6 +120,139 @@ function appendUnlabeledStat(table, text) {
 	return [tr, textCell];
 }
 
+
+// appendNavigationButtons appends two buttons that navigate between the previous and the next block.
+/* function appendNavigationButtons(element, explorerBlock) {
+	var buttonContainer = document.createElement('div');
+	buttonContainer.classList.add('navigation-buttons');
+
+	var previousButton = document.createElement('button');
+	previousButton.id = 'previousbutton';
+	previousButton.textContent = 'Previous Block';
+	if (explorerBlock.height == 0) {
+		previousButton.disabled = true;
+	}
+
+	var nextButton = document.createElement('button');
+	nextButton.id = 'nextbutton';
+	nextButton.textContent = 'Next Block';
+
+	buttonContainer.appendChild(nextButton);
+	buttonContainer.appendChild(previousButton);
+	element.appendChild(buttonContainer);
+
+	previousButton.onclick = (e) => {
+		goToPreviousBlock(explorerBlock.height);
+	}
+	
+	nextButton.onclick = (e) => {
+		goToNextBlock(explorerBlock.height);
+	}
+
+} */
+
+function appendNavigationElements(element, explorerBlock) {
+	//create all needed elements
+	var buttonContainer = document.createElement('div');
+	var searchButtonForm = document.createElement('form');
+	var previousButton = document.createElement('button');
+	var nextButton = document.createElement('button');
+	var searchButton = document.createElement('button');
+	var searchField = document.createElement('INPUT'); 
```

Comment:
please put validation on this field:

* it should only accept numerical characters (`[0-9]`);
* it can only be positive (should already be clear because of first validation rule);